docs/content/patterns/alz/deploy/Deploy-only-Service-Health-Alerts.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Deploy only Service Health Alerts
+geekdocCollapseSection: true
+weight: 70
+---
+
+The following guide describes the steps to leverage the ALZ pattern to implement Service Health Alerts. When you deploy one initiative, like Service Health, you will only need the Policy Definitions required by that Initiative. You can still choose to deploy all Policy Definitions that are provided in the ALZ Pattern, this is recommended when you want to deploy other Initiatives in the future. In case you first deploy a subset of the Policy Definitions, you can easily deploy additional definitions at a later stage. This document covers two deployment options:
+
+1. [Quick Deployment](../Deploy-only-Service-Health-Alerts/#quick-deployment): Deploys the ALZ Pattern including all Policy Definitions, Policy Set Definitions, however, this assigns only the Service Health initiative.
```

Comment:
Can we use the same language throughout Policy Set Definition or Policy Initiative